src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Guid.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Buffers.Text;
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    public ref partial struct Utf8JsonWriter
+    {
+        /// <summary>
+        /// Writes the property name and <see cref="Guid"/> value (as a JSON string) as part of a name/value pair of a JSON object.
```

Comment:
I think it is odd to choose to make Guid overloads first-class methods but whatever 🤷‍♂️ 

There is no accepted way to represent a Guid in a JSON string (dashes vs no dashes, lowercase vs uppercase, etc) so add info to comment.

Reply:
Yep, will do. @bartonjs already provided such feedback: https://github.com/dotnet/corefx/pull/34425#discussion_r246077000

We format based on whatever the default `Utf8Formatter` behavior is (when no StandardFormat is specific). In this case, it is D (default): nnnnnnnn-nnnn-nnnn-nnnn-nnnnnnnnnnnn